### DISCUSSION TITLE
git remote add origin paste_github_url_link

### DISCUSSION THREAD

**Discussion by Sukankrishnan** [Vish-devop/JavaScript_Learning#2] [Q&A] [answered]
When I tried to run this command, I got warning messages that permission was denied. Am I using the wrong path?

**Commands I tried:**

Your environment has been set up for using Node.js 20.11.0 (x64) and npm.

C:\Users\sukan>git init
Initialized empty Git repository in C:/Users/sukan/.git/

C:\Users\sukan>git remote add origin https://github.com/Sukankrishnan/MyJavascriptLearning.git

C:\Users\sukan>git add .
warning: could not open directory 'AppData/Local/Application Data/': Permission denied
warning: could not open directory 'AppData/Local/History/': Permission denied
warning: could not open directory 'AppData/Local/Microsoft/Windows/INetCache/Content.IE5/': Permission denied

**Comment by Sukankrishnan** [comment, marked as answer]
It gets resolved when I use GitBash as the above commands are related to GitBash

**Reply by Vish-devop** [reply]
Great!
